app/lib/features/space/widgets/chats_card.dart

```diff
@@ -49,12 +50,19 @@ class ChatsCard extends ConsumerWidget {
                       /// FIXME: push is broken for switching from subshell to subshell
                       /// hence we are using `go` here.
                       /// https://github.com/flutter/flutter/issues/125752
-                      onTap: () => context.goNamed(
-                        Routes.chatroom.name,
-                        pathParameters: {
-                          'roomId': chats[index].getRoomIdStr(),
-                        },
-                      ),
+                      onTap: () => isDesktop
+                          ? context.goNamed(
+                              Routes.chatroom.name,
+                              pathParameters: {
+                                'roomId': chats[index].getRoomIdStr(),
+                              },
+                            )
+                          : context.pushNamed(
+                              Routes.chatroom.name,
+                              pathParameters: {
+                                'roomId': chats[index].getRoomIdStr(),
+                              },
+                            ),
```

Comment:
hmm... are we sure we want this? The fixme is relating to the subshells and we are getting rid of them ... I think this might be superflous if we also removed the chat-subshell.

Reply:
- Yes sure, I will remove fixme comment
- Added routing condition temporary till finish removal of the chat-subshell. Otherwise sometime it is miss behaving on desktop if we use pushNamed.